dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 black
 graphviz >= 0.8
 jinja2 >= 2.0, < 4.0
-mypy >= 0.600, < 0.813
```

Comment:
We're pinning below this version because we don't have separate typeshed installations. Is there a reason you bumped this?

Reply:
I'm not sure why I did this.
Probably not intentional.
Anyway, I should have reverted back to `mypy >= 0.600, 0.813`